backend/src/rss.rs

```diff
@@ -55,19 +56,26 @@ pub(crate) async fn generate_rss_feed(context: &Arc<Context>, id: &str) -> Resul
                     xmlns:dc="http://purl.org/dc/elements/1.1/"
                     xmlns:atom="http://www.w3.org/2005/Atom"
                     xmlns:media="http://search.yahoo.com/mrss/"
+                    xmlns:itunes="http://www.itunes.com/dtds/podcast-1.0.dtd"
+                    xmlns:content="http://purl.org/rss/1.0/modules/content/"
                 >
                 <channel>
                     <title>{}</title>
                     <link>{}</link>
                     <description>{}</description>
+                    <language>und</language>
                     <atom:link href="{}" rel="self" type="application/rss+xml" />
+                    <itunes:category text="Education" />
```

Comment:
Yeah I agree, but this tag as well as the `<itunes:explicit>` one are required by itunes ([ref](https://help.apple.com/itc/podcasts_connect/#/itcb54353390)) and I don't think there is any way this could be inferred from the series data we have. Omitting these wouldn't break the rss but quoting the link above: `Required tags must be present in your RSS feed or it won’t pass validation to be listed in Apple Podcasts.`, so Tobira's RSS feeds wouldn't be listed in apple podcasts... which I think is fine? But honestly no idea, @oas777 specifically mentioned itunes validation in his [comment](https://github.com/elan-ev/tobira/issues/791#issuecomment-1534405300) on the issue, which is the reason I included the `itunes` tags in the first place.

Reply:
> And for the category, is there a super generic one maybe?

I don't think so. Assuming [this](https://podcasters.apple.com/support/1691-apple-podcasts-categories) features an exhaustive list, the only ones that could be considered as generic and somewhat fitting for Tobira series are `Science` and super duper maybe `Society & Culture`, depending on who you ask. I think `Education` really is the most fitting, even if it might not apply to every video.

Reply:
Huff okay, yeah "Education" then I suppose.